website/content/configuration/troubleshooting.md

```diff
@@ -5,14 +5,17 @@ weight: 20
 
 Once you setup a loadbalancer on Kubernetes and it doesn't _seem_ to work right away, here are a few tips to troubleshoot.
 
-SSH into one or more of your nodes and use `arping` and `tcpdump` to verify the ARP requests pass through your network.
+- Check that the LoadBalancer service has endpoints (`kubectl get endpoints`) that are not pending - if they are, MetalLB will not respond to ARP requests for that service's external IP
+- SSH into one or more of your nodes and use `arping` and `tcpdump` to verify the ARP requests pass through your network
+
```

Comment:
note to self: I see an extra line here. I'd need to check, just in case, if this doesn't break the hugo formatting

Reply:
I added a `---` (hr) after to break up the content a bit, from my experience Markdown likes an extra line between disparate elements, seemed to generate properly when I tested the changes with Hugo.

Reply:
![image](https://user-images.githubusercontent.com/32423458/78471751-882a6400-772b-11ea-88c2-5d4fcee6b09c.png)

Reply:
Thanks!